Core/Date.cs

```diff
@@ -30,315 +30,138 @@ namespace System
 
-		public int DayOfYear
-		{
-			get
-			{
-				return _dt.DayOfYear;
-			}
-		}
+		public int DayOfYear => _dt.DayOfYear;
 
-		public int Month
-		{
-			get
-			{
-				return _dt.Month;
-			}
-		}
+		public int Month => _dt.Month;
 
-		public static Date Today
-		{
-			get
-			{
-				return new Date(DateTime.Today);
-			}
-		}
+		public static Date Today => new Date(DateTime.Today);
 
-		public int Year
-		{
-			get
-			{
-				return _dt.Year;
-			}
-		}
+		public int Year => _dt.Year;
 
-		public long Ticks
-		{
-			get
-			{
-				return _dt.Ticks;
-			}
-		}
+		public long Ticks => _dt.Ticks;
 
-		public Date AddDays(int value)
-		{
-			return new Date(_dt.AddDays(value));
-		}
+		public Date AddDays(int value) => new Date(_dt.AddDays(value));
 
-		public Date AddMonths(int value)
-		{
-			return new Date(_dt.AddMonths(value));
-		}
+		public Date AddMonths(int value) => new Date(_dt.AddMonths(value));
 
-		public Date AddYears(int value)
-		{
-			return new Date(_dt.AddYears(value));
-		}
+		public Date AddYears(int value) => new Date(_dt.AddYears(value));
 
-		public static int Compare(Date d1, Date d2)
-		{
-			return d1.CompareTo(d2);
-		}
+		public static int Compare(Date d1, Date d2) => d1.CompareTo(d2);
 
-		public int CompareTo(Date value)
-		{
-			return _dt.CompareTo(value._dt);
-		}
+		public int CompareTo(Date value) => _dt.CompareTo(value._dt);
 
-		public int CompareTo(object value)
-		{
-			return _dt.CompareTo(value);
-		}
+		public int CompareTo(object value) => _dt.CompareTo(value);
 
-		public static int DaysInMonth(int year, int month)
-		{
-			return DateTime.DaysInMonth(year, month);
-		}
+		public static int DaysInMonth(int year, int month) => DateTime.DaysInMonth(year, month);
 
-		public bool Equals(Date value)
-		{
-			return _dt.Equals(value._dt);
-		}
+		public bool Equals(Date value) => _dt.Equals(value._dt);
 
-		public override bool Equals(object value)
-		{
-			return value is Date && _dt.Equals(((Date)value)._dt);
-		}
+		public override bool Equals(object value) => value is Date && _dt.Equals(((Date)value)._dt);
 
-		public override int GetHashCode()
-		{
-			return _dt.GetHashCode();
-		}
+		public override int GetHashCode() => _dt.GetHashCode();
 
-		public static bool Equals(Date d1, Date d2)
-		{
-			return d1._dt.Equals(d2._dt);
-		}
+		public static bool Equals(Date d1, Date d2) => d1._dt.Equals(d2._dt);
 
-		void ISerializable.GetObjectData(SerializationInfo info, StreamingContext context)
-		{
-			info.AddValue("ticks", _dt.Ticks);
-		}
+		void ISerializable.GetObjectData(SerializationInfo info, StreamingContext context) => info.AddValue("ticks", _dt.Ticks);
 
-		public static bool IsLeapYear(int year)
-		{
-			return DateTime.IsLeapYear(year);
-		}
+		public static bool IsLeapYear(int year) => DateTime.IsLeapYear(year);
 
-		public static Date Parse(string s)
-		{
-			return new Date(DateTime.Parse(s));
-		}
+		public static Date Parse(string s) => new Date(DateTime.Parse(s));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the overload that takes a 'CultureInfo' or 'IFormatProvider' parameter.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4931949)